amu_roboclub_team.json

```diff
@@ -0,0 +1,328 @@
+    "current" : {
+    	"info" : {
+    		"session" : "2019-20"
+    	},
+    	"members" : {
+    		"abidalikhan" : {
+    			"name" : "Dr. Abid Ali Khan",
+    			"position" : "Faculty Adviser",
+    			"rank" : 20,
+    			"thumbnail" : "https://res.cloudinary.com/amuroboclub/image/upload/old/team/DrAbidAliKhan.bmp"
+    		},
+    		"anwarsadat" : {
+    			"links" : {
+    				"email" : "anwar7039@gmail.com"

```

Comment:
Why is there a problem in copy/pasting?

You changed it from correct email to wrong one

Reply:
I didn't copy paste and it's the same email in the file which Haider sent me. Under the 2018-19 session.

Reply:
I didn't copy paste.

Reply:
That was the problem. Why didn't you copy/paste. These are not the things you write by hand